index.js

```diff
@@ -0,0 +1,177 @@
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
Overall good working app with few feedbacks: 
1. variable names could be shorter and also could be consolidated into fewer variables that are essential. 
2. Things could be put in functions to make it more readable and streamlined. 
3. Lots of nested "if" statements which means logics could also be simplified a little. 
Other than above mentioned comments, app is working as intended which is good. Keep up the good work and looking forward to see what you will have for next projects.